.travis.yml

```diff
@@ -20,6 +20,8 @@ matrix:
           env: DEPLOY=yes TASK_TESTS_COVERAGE=1
         - php: 7.1
           env: TASK_SCA=1
+        - php: nightly
```

Comment:
[nope](https://secure.php.net/supported-versions.php), so lets wait for official release to set the release version, PHP might just jump another major to 9 given its history ;)